pkg/local-gadget-manager/local-gadget-manager_test.go

```diff
@@ -647,8 +647,35 @@ func TestNetworkGraph(t *testing.T) {
 		},
 		PktType: "OUTGOING",
 		Proto:   "tcp",
-		Addr:    "1.1.1.1",
-		Port:    443,
+		// There's no place like
```

Comment:
I don't get this comment.

Reply:
The line below is 127.0.0.1, aka the 'home' IP. The comment says 'There's no place like home'.

Reply:
I haven't ever heard about it 😅 . I have always called it localhost.